build.sbt

```diff
@@ -7,14 +7,14 @@ lazy val gcTask = gc := {
   System gc()
 }
 
-lazy val project = Project("project", file("."))
+lazy val root = project.in(file("."))
```

Comment:
The variable name here is irrelevant and idiomatically all sbt projects use the `project.in` syntax